### PR TITLE
Fix #1564, replace Blake with Keccak

### LINKED ISSUE #1564: Change Hashing Function from Blake2b to Sha3
### What is wrong?

Per https://github.com/ethereum/eth2.0-specs/pull/227

### How can it be fixed

1. Use this one:
    ```
    from eth_hash.auto import keccak
    ```
2. Remove `eth/utils/blake.py` and related tests.

### PR DESCRIPTION
**Pull request by ChihChengLiang**
### What was wrong?

Keccak is the hash function for Beacon chain now and will be replaced in the future.

### How was it fixed?

- Wrap `keccak` with a function `hash_`. This ensures the future switch would be fast. 
- Place `hash_` in `eth/beacon/utils/hash.py`, since `hash.py` is only relevant to beacon chain.
- Also fixed some tests that depend on Blake.

#### Cute Animal Picture

put a cute animal picture link inside the parentheses
By James Rosindell - Own work, CC BY-SA 4.0, https://commons.wikimedia.org/w/index.php?curid=45692802

### CODE REVIEW

File: tests/core/hash-utils/test_hash.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from eth.beacon.utils.hash import hash_
+
+
+def test_hash():
+    output = hash_(b'helloworld')
+    assert len(output) == 32
```
Comment (hwwhww) [inline tests/core/hash-utils/test_hash.py]: Maybe add a test for `test_hash_is_keccak256()`?

File: tests/beacon/utils/test_random.py [hunk 2]
```diff
@@ -11,16 +11,24 @@
     ),
     [
         (
-            [i for i in range(100)],
-            b"\x8d\x8dd\xb9\x84\x05*Bw\xf5\x87\xdd\x04nj\xee\x82\xfc9\xb9\x19\x82'x!#p];\xa5\x8e\xc9",  # noqa: E501
-            (87, 32, 76, 41, 61, 9, 44, 78, 70, 42, 8, 6, 79, 37, 4, 67, 3, 66, 55, 0, 69, 98, 38, 47, 20, 31, 48, 97, 34, 33, 96, 63, 91, 15, 1, 77, 39, 64, 58, 53, 82, 49, 26, 29, 5, 65, 19, 86, 74, 23, 46, 14, 10, 51, 85, 89, 57, 73, 71, 52, 88, 30, 16, 81, 24, 22, 36, 45, 12, 2, 62, 93, 18, 72, 83, 95, 13, 21, 27, 28, 80, 43, 68, 59, 94, 60, 99, 56, 92, 54, 25, 75, 7, 90, 50, 35, 11, 84, 17, 40),  # noqa: E501
+            tuple(range(100)),
+            b'\x32' * 32,
+            (92, 18, 80, 86, 39, 56, 81, 74, 68, 64,
```
Comment (hwwhww) [inline tests/beacon/utils/test_random.py]: formatting
```python
(
    92, 18, 80, 86, 39, 56, 81, 74, 68, 64,
    ....
    67, 96, 78, 8, 10, 99, 87, 27, 32, 23
),

```

File: tests/beacon/utils/test_random.py [hunk 3]
```diff
@@ -11,16 +11,24 @@
...
         ),
         (
-            [i for i in range(12)],
-            b"&P\x18'\x91\xf6\x8f\xc8;<\xb8\x8f\x1d\x92?\xad4\xb5\xb8\x0b\xca\x8a'\x8e\xae'\xcf7\xa3(\xb2\x8d",  # noqa: E501
-            (7, 3, 2, 5, 11, 9, 1, 0, 4, 6, 10, 8),
-
-        )
+            tuple(range(12)),
+            b'\x23' * 32,
+            (11, 4, 9, 5, 7, 10, 2, 8, 0, 6, 3, 1),
```
Comment (hwwhww) [inline tests/beacon/utils/test_random.py]: This test was from lighthouse's codebase. Maybe open an issue to try out https://github.com/status-im/eth2-testgen/blob/master/test_vectors/test_vector_shuffling.yml.

Comment (ChihChengLiang) [reply tests/beacon/utils/test_random.py]: issue opened here https://github.com/ethereum/py-evm/issues/1615

**Comment by hwwhww** [comment]
link #1564

**Pull request merged**